cmd/config.go

```diff
@@ -50,8 +51,8 @@ func (conf *mqttConfig) RootTopic() string {
 }
 
 type qualifiedConfig struct {
-	Name, Type string
-	Other      map[string]interface{} `mapstructure:",remain"`
+	Name, Type, Template string
```

Comment:
Siehe nächster Kommentar

Reply:
Welcher nächster Kommentar? Da steht "s.o.". Möchte mich ungern hier im Kreis zwischen den Kommentaren drehen :D

Reply:
Ich meine wir brauchen das Template erst in NewfromConfig, aber nicht hier! Das leakt Implementierungsdetails die hier irrelevant sind.

Reply:
aehm. und wie kommt dann `template: sma-hm` aus der Konfig hier rein? Wenn es da nicht drin ist, kann ich das in der Konfig Datei nicht verwenden

Reply:
In der Konfigurationsdatei steht ja so etwas:

```
meters:
- template: sma-home-manager
  usage: grid 
  host: 192.168.1.2
```

Damit das funktioniert, muss `Template` hier in den struct rein. Nochmal, in der Konfig soll so etwas knappes / kurzes stehen, nicht das komplizierte was es ersetzt.

Reply:
Rein ja, hier nein. Das gehört in newtemplatefromconfug, da muss nochmal ein decode passieren um das template vom rest zu trennen. Dass es template gibt ist erst und nur dort relevant.

Reply:
Das passt schon alles, erscheint mir aber noch zu global umgesetzt zu sein

Reply:
Done